source/develop/release-management/process/making-an-release.md

```diff
@@ -2,24 +2,171 @@
+In order to do that we are going to use the [CentOS Community Build Service (CBS)](https://cbs.centos.org/koji/).
+
+Package maintainers must have a [CentOS Account](https://accounts.centos.org/) or a [Fedora Account](https://accounts.fedoraproject.org/),
+which were unified in the middle of 2021 to use the same account system.
+
+In order to build for oVirt you need to be added to the CBS you need to join the [CentOS Virtualization SIG](https://wiki.centos.org/SpecialInterestGroup/Virtualization).
+You can do that by attending a [Virt SIG IRC meeting](https://www.centos.org/community/calendar/#virtualization-sig)
+or asking on the [CentOS Virt mailing list](https://lists.centos.org/mailman/listinfo/centos-virt)
+
+> Please note that if you want to ship oVirt packages in other distributions you'll need to get permissions there.
+> For example, if you want to ship for Fedora, you need to be package owner on [Fedora Build System](https://koji.fedoraproject.org/koji/).
+>
+> At the time of writing, the oVirt project is focusing only on [CentOS Stream](https://www.centos.org/centos-stream/).
+
+The build process on the [Community Build Service is documented on the CentOS website](https://wiki.centos.org/HowTos/CommunityBuildSystem).
+
+A typical build is done by taking a src.rpm from the COPR [ovirt/ovirt-master-snapshot Builds](https://copr.fedorainfracloud.org/coprs/ovirt/ovirt-master-snapshot/builds/)
+and rebuilding with:
+
+```bash
+# This builds a scratch (just a test) build for CentOS Stream 9 for oVirt 4.5
+cbs build --scratch virt9s-ovirt-45-el9s <your package .src.rpm>
+
+# This is for CentOS Stream 8:
+cbs build --scratch virt8s-ovirt-45-el8s <your package .src.rpm>
+
+# if the build goes fine you can build for real
+# CentOS Stream 9:
+cbs build virt9s-ovirt-45-el9s <your package .src.rpm>
+
+# CentOS Stream 8:
+cbs build virt8s-ovirt-45-el8s <your package .src.rpm>
+```
+
+Once the builds complete it will be tagged as a candidate to be released as `virt8s-ovirt-45-candidate` (for CentOS Stream 8) or `virt9s-ovirt-45-candidate` (for CentOS Stream 9).
+
+In order to publish the build you need to tag it for testing:
+
+```bash
+# CentOS Stream 8
+cbs tag virt8s-ovirt-45-testing <your build without the trailing '.src.rpm'>
+
+#example:
+cbs tag virt8s-ovirt-45-testing ovirt-hosted-engine-ha-2.4.10-1.el8
+```
+
+this will publish the build on https://buildlogs.centos.org/centos/8-stream/virt/ repository.
+
+
+```bash
+# CentOS Stream 9
+cbs tag virt9s-ovirt-45-testing <your build without the trailing '.src.rpm'>
+
+#example:
+cbs tag virt9s-ovirt-45-testing ovirt-hosted-engine-ha-2.4.10-1.el9
+```
+
+this will publish the build on https://buildlogs.centos.org/centos/9-stream/virt/ repository.
+
+
+## Release phase
+
+Once the content in testing has been properly verified and considered good to be released for general availability you'll need to tag your builds for release:
```

Comment:
Does this really say that every maintainer will need to tag their respective RPMs after the build passes QE? If this will be done by single person (and serves only documentation purposes), this should be reworded to avoid confusion.  The roles of maintainers and release manager should be clearly separated.

Reply:
re-phrased to clarify this needs to be done by release manager